pkg/gits/git_cli.go

```diff
@@ -379,49 +379,37 @@ func (g *GitCLI) CommitIfChanges(dir string, message string) error {
 
 // GetCommits returns the commits in a range, exclusive of startSha and inclusive of endSha
 func (g *GitCLI) GetCommits(dir string, startSha string, endSha string) ([]GitCommit, error) {
-	args := []string{"log", "--format=raw", fmt.Sprintf("%s..%s", startSha, endSha)}
+	// use a custom format to get commits, using %x1e to separate commits and %x1f to separate fields
+	args := []string{"log", "--format=%H%x1f%an%x1f%ae%x1f%cn%x1f%ce%x1f%s%n%b%x1e", fmt.Sprintf("%s..%s", startSha, endSha)}
 	out, err := g.gitCmdWithOutput(dir, args...)
 	if err != nil {
 		return nil, errors.Wrapf(err, "running git %s", strings.Join(args, " "))
 	}
 	answer := make([]GitCommit, 0)
-	commits := strings.Split(out, "commit ")
```

Comment:
@pmuir, I don't think splitting at 'commit' is going to work. It breaks if 'commit' appears in the message, eg merge commits.